vampyr/operators/convolutions.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include <MRCPP/operators/HelmholtzOperator.h>
+#include <MRCPP/operators/IdentityConvolution.h>
+#include <MRCPP/operators/PoissonOperator.h>
+
+namespace vampyr {
+
+void helmholtz_operator(pybind11::module &);
+void poisson_operator(pybind11::module &);
+
+template <int D> void convolutions(pybind11::module &m) {
+    namespace py = pybind11;
+    using namespace mrcpp;
+    using namespace pybind11::literals;
+
+    py::class_<ConvolutionOperator<D>>(m, "ConvolutionOperator")
+        .def(py::init<const MultiResolutionAnalysis<D> &, double>());
+
+    py::class_<IdentityConvolution<D>, ConvolutionOperator<D>>(m, "IdentityConvolution")
+        .def(py::init<const MultiResolutionAnalysis<D> &, double>(),
+             "mra"_a,
+             "prec"_a);
+
+    if (D==3) helmholtz_operator(m);
+    if (D==3) poisson_operator(m);
```

Comment:
I guess this can be `constexpr if`, if you compile with C++17.

Reply:
hmm, the GHA builds show both `-std=c++14` and `-std=c++17` :thinking: 
```
-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell
-ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem
/usr/share/miniconda/envs/vampyr-gha/include -march=native -O3 -DNDEBUG -fPIC -fopenmp
-std=c++14 -MD -MT 
```

Reply:
The build happens in a conda environment so it inherits whatever `CXXFLAGS` conda feels like. Try doing:
```
    - name: Build
      shell: bash -l {0}
      env:
        CXXFLAGS: ""
      run: |
        cmake --build build --config $BUILD_TYPE --target install -- -v -d stats
```
in the workflow file.

Reply:
No luck :disappointed:

Reply:
You need to set the standard in one CMake file (done now)

Reply:
We still get `-std=c++17` and `-std=c++14` as well as `-O2` and `-O3`

Reply:
Uff. It's also linking against gcov.